pkg/gitproviders/provider_test.go

```diff
@@ -38,7 +38,7 @@ type customTransport struct {
 
 func getBodyFromReaderWithoutConsuming(r *io.ReadCloser) string {
 	body, _ := ioutil.ReadAll(*r)
```

Comment:
This is out of the scope of this PR, and wasn't changed, but probably because this is just a method called from a custom roundtrip implementation inside a test, and it's very unlikely it will every error out from reading a request body from memory.

Reply:
Ah, I missed that it was in a test file.  Thanks!